packages/backend/src/peripherals/database/migrations/058_block_numbers_chain_id.ts

```diff
@@ -0,0 +1,40 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { ChainId } from '@l2beat/shared-pure'
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.alterTable('block_numbers', function (table) {
+    table
+      .string('chain_id')
```

Comment:
why string? Can't we have integers like everyone? 1 is ethereum, 42 optimism i think and so on.